tests/test_uncrustify/config.py

```diff
@@ -0,0 +1,36 @@
+# Global state.
+#
+# * @author  Ben Gardner        October 2009
+# * @author  Guy Maurel         October 2015
+# * @author  Matthew Woehlke    June 2018
+#
+
+import os
+
+test_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+
+all_tests = [
```

Comment:
can't we just replace this with glob *.test?

Reply:
Well... the old behavior was to list the suites explicitly. AIUI, this is because there might be other `.test` files that should *not* be run by default, but that can be run by the user by specifying the name. I suppose if we want to change that behavior, we could? At any rate, I'm just preserving what we did before, which was apparently intentional.

Reply:
ok, let's keep the old behavior.